bindings/haskell/if-proto-scip-hs.cabal

```diff
@@ -0,0 +1,21 @@
+cabal-version:      3.6
+
+name:               if-proto-scip-hs
+version:            0.1.0.0
+synopsis:           Haskell bindings for SCIP Code Intelligence Protocol
+license:            Apache-2.0
+license-file:       LICENSE
+author:             SCIP Maintainers
+maintainer:         code-intel@sourcegraph.com
+homepage:           https://github.com/sourcegraph/scip
+category:           Language
+build-type:         Simple
+
+library
+    ghc-options: -Wall
+    exposed-modules:
+        Proto.Scip,
+        Proto.Scip_Fields
```

Comment:
Can we tweak the generation so that the names for the modules are `SCIP` and `SCIPFields`? That would match existing Haskell conventions better around acronyms.

Reply:
I've manually renamed these for now.